### DISCUSSION TITLE
Фильтр по полю с типом чекбокс

### DISCUSSION THREAD

**Discussion by bachinin** [totumonline/totum-ru-issues-and-discussions#602] [Как это сделать?] [answered]
Здравствуйте.
В таблице есть поле с типом чекбокс.  Если добавляю префильтр по этому полю (тоже с типом чекбокс), то таблица становится всегда зафильтрованной: либо true, либо false. Есть ли какая-то возможность сделать так, чтобы в префильтре помимо true и false можно было выбрать значение ВСЕ?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
https://ru.docs.totum.online/using-manual-selections-in-prefilters
